ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -113,7 +156,7 @@ void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool *stopPropagati
     RNS_LOG_DEBUG("onfocus need to here"<<textInputMetrics.text);
     textInputEventEmitter->onFocus(textInputMetrics);
     isInEditingMode_ = true;
-  } else if (isInEditingMode_) {
+  } else if ((isInEditingMode_)  && (textInputProps.editable)) {
```

Comment:
with this check , we will be sending onFocus event and setting isInEditingMode to true . But there will no onBlur event sent and isInEditingMode is never set to false .
Please confirm is onFocus event need to be sent if editable is true. If yes , then we have to ensure to send onBlur event as well.